### PR TITLE
frontsite: DocumentDetails: Display multipart above Literature Title

### PR DESCRIPTION
**Pull request by sakshamarora1**
:heart: Thank you for your contribution!

### Description

closes: https://github.com/CERNDocumentServer/cds-ils/issues/734

Screenshot 2024-06-28 at 15 12 40

Screenshot 2024-06-28 at 15 13 10

### CODE REVIEW

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js [hunk 1]
```diff
@@ -19,12 +21,38 @@ export class DocumentTitle extends Component {
...
+    ) : null;
+  }
+
   render() {
     const { metadata } = this.props;
     const volume = _get(metadata, 'relations.multipart_monograph[0].volume');
 
     return (
       <>
+        <Header>
```
Comment (kpsherva) [inline src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js:51]: if you don't check if the volume number and relation metadata exists before rendering the Header component, you will most likely end up with an empty space on top of the column (for regular books without multipart and serials)

File: src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js [hunk 2]
```diff
@@ -19,12 +21,38 @@ export class DocumentTitle extends Component {
...
+
+  relationsTitle(relationsData) {
+    return !_isEmpty(relationsData) ? (
+      <Header.Subheader>
+        {relationsData.map((rel, index) => (
+          <>
+            <Link key={rel.pid_value} to={this.getLinkTo(rel)}>
+              {rel.record_metadata.title}
+            </Link>
+            {index === relationsData.length - 1 ? ' featuring:' : ' and '}
```
Comment (kpsherva) [inline src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js:38]: not sure I understand why we need "and"?

Comment (sakshamarora1) [reply src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js:38]: Because it is a list, there can be multiple relations

Comment (kpsherva) [reply src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js:38]: normally, a volume can belong only to one multipart monograph (but multiple serials, that's why it is modelled as a list), but for this particular case we should display only one multipart

**Comment by ntarocco** [comment]
The PR looks good, but I have some doubts on the UX/design of the `featuring:` part. To me, is not visible and not very understandable.
Screenshot 2024-05-27 at 09 37 51

File: src/semantic-ui/site/globals/site.overrides [hunk 3]
```diff
@@ -357,6 +357,10 @@ a {
     color: #1aa3ff;
   }
 
+  a.relations-title{
+    color: black;
```
Comment (kpsherva) [inline src/semantic-ui/site/globals/site.overrides:361]: I would leave it blue, as a link, and have it as cursive instead

**Pull request merged**